code/public/index.html

```diff
@@ -3,17 +3,21 @@
 
   <head>
     <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no,initial-scale=1.0001,maximum-scale=1.0001, viewport-fit=cover" />
 
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-    <title>Technigo React App</title>
+
+    <!--Favicon-->
+    <link rel="icon" type="image/png" href="black-pen.png">
+
+
+    <!-- OG TAGS – SOCIAL MEDIA -->
+    <meta property="og:title" content="Emma Lindell – Todo App" />
+    <meta property="og:description" content="Add something to your very own todo-list" />
+    <meta property="og:image" content="https://i.postimg.cc/nz6yFywn/todo-app.png" />
+    <meta property="og:url" content="https://dothistoday.netlify.app/" />
+    </head>
```

Comment:
I like the use of OG tags in your project! I tried it out and it seem to work :)